src/app/messages/en/attributionText.ts

```diff
@@ -55,77 +60,10 @@ export const defaultText = `
 </ul>
 
 <p>
-  © {bookLatestRevision, date, medium} OpenStax. Textbook content produced by OpenStax is licensed under a
-  {bookLicenseName} {bookLicenseVersion} license. <strong>The OpenStax name, OpenStax logo, OpenStax book
-  covers, OpenStax CNX name, and OpenStax CNX logo are not subject to the Creative Commons license and may
-  not be reproduced without the prior and express written consent of Rice University.</strong>
-</p>
-`;
-
-export const teaText = `
-<p>
-  Want to cite, share, or modify this book? This book is
-  <a target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by/4.0/">
```

Comment:
the TEA physics book requires that this link be here. we can add the link unconditionally to the unified attribution message but we have to use the license link out of the archive data `book.license.url`